### PR TITLE
add different types of Alerts and Buttons

### PR DESCRIPTION
**Pull request by hsnice16**
Added components for
+ Alerts
   - Danger Alert
   - Info Alert
   - Primary Alert
   - Secondary Alert
   - Success Alert
   - Warning Alert
   
+ Buttons
    - Filled Primary Buttons
    - Outlined Primary Buttons
    - Link Button
    - Icon Button
    - Floating Button
    
 Now, File structure is

```
  /
  |
  | -- css /
  |       | -- components / 
  |       |         | -- Alert /
  |       |         |       | -- alert.css
  |       |         | -- Button /
  |       |                 | -- button.css
  |       |          
  |       | -- utilities /
  |       |        | -- background.css
  |       |        | -- margin.css
  |       |
  |       | -- main.css
  |       | -- root.css
  |
  | -- README.md
  | -- app.css
  | -- index.html
 ```

To see UI of the components, Check [https://poshui-component-alert-button.netlify.app/#](https://poshui-component-alert-button.netlify.app/#)

Please review this PR, and give your feedback.

Thankyou for reviewing it. 🙏🏻

### CODE REVIEW

**Review by animeshgarai09 — COMMENTED** [review]
Class names can be improved.

**Comment by hsnice16** [comment]
Thanks @animeshgarai09, Can you suggest something? and where I can improve them? Or by adding comments only It can work?

File: css/components/Alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
+.alert {
+  border-radius: 0.5rem;
+  font-weight: bold;
+  margin: 1rem 0;
+  max-width: 25rem;
+  padding: 1rem;
+}
+
+.alert-f-danger {
```
Comment (venkateshwarans) [inline css/components/Alert/alert.css]: does f mean fill in `.alert-f-danger`?  Maybe you can use `alert-danger-solid` `alert-danger-outline`.

Comment (hsnice16) [reply css/components/Alert/alert.css]: Yeah, it means `fill`. Thanks, I will change them.

File: css/components/Button/button.css [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+/** Button Squared **/
+
+.btn-squared {
+  border-radius: 1rem;
+}
+
+/** Filled Buttons **/
+
+.btn-f-error {
```
Comment (venkateshwarans) [inline css/components/Button/button.css]: here as well - .btn-error-fill or .btn-error-solid . choose what sounds better to you.

**Review by venkateshwarans — APPROVED** [review]
Apart from the CSS class name, the rest looks great.

File: css/components/Button/button.css (hunk 2 above)
Comment (hsnice16) [reply css/components/Button/button.css]: I was changing here too, 😅. To keep consistency, I have decided to use `btn-error-solid`. 

This would be good ?

Comment (venkateshwarans) [reply css/components/Button/button.css]: Yeah. 👍

**Comment by hsnice16** [comment]
@animeshgarai09  and @venkateshwarans, if you can review now. I have changed class names, and there is some changes in File Structure too.

Now,
```
 /
  |
  | -- css /
  |       | -- components / 
  |       |         | -- Alert /
  |       |         |       | -- alert.css
  |       |         | -- Button /
  |       |                 | -- button.css
  |       |                 | -- outline.css
  |       |                 | -- solid.css
  
      ...
```

**Pull request merged**